lib/proxy_api/tftp.rb

```diff
@@ -1,27 +1,28 @@
 module ProxyAPI
   class TFTP < ProxyAPI::Resource
     def initialize(args)
-      @url     = args[:url] + "/tftp"
-      @variant = args[:variant]
```

Comment:
Why not renaming `variant` to kind and leaving it as is ? It requires less changes to the class